python/ray/tune/impl/tuner_internal.py

```diff
@@ -253,23 +261,18 @@ def _validate_overwrite_trainable(self, original_trainable, overwrite_trainable)
                 )
 
         logger.warning(
```

Comment:
warnings.warn here as well?

Reply:
I kept this one as a logged warning, since it'll always be logged when the trainable gets overwritten, and there's no user action that suppresses the warning. I referenced [this post](https://stackoverflow.com/a/40667761/13232243).

Reply:
Sg, thanks for clarifying!